velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -1132,13 +1136,20 @@ class ArrowBridgeArrayImportTest : public ArrowBridgeArrayExportTest {
     }
 
     holder.nulls = AlignedBuffer::allocate<uint64_t>(length, pool_.get());
-    holder.offsets = AlignedBuffer::allocate<int32_t>(length + 1, pool_.get());
+    holder.offsets = is32
+        ? AlignedBuffer::allocate<int32_t>(length + 1, pool_.get())
+        : AlignedBuffer::allocate<int64_t>(length + 1, pool_.get());
     holder.values = AlignedBuffer::allocate<char>(bufferSize, pool_.get());
 
     auto rawNulls = holder.nulls->asMutable<uint64_t>();
     auto rawOffsets = holder.offsets->asMutable<int32_t>();
+    auto rawOffsets64 = holder.offsets->asMutable<int64_t>();
```

Comment:
Looks like we can always use rawOffsets but set it to holder.offsets->asMutable<int64_t>() or holder.offsets->asMutable<int32_t>(), it does not happen at the same time.